recipes/handsontable/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "handsontable" %}
+{% set version = "0.34.5" %}
+{% set sha256 = "476e42334b0a1e6b810018a588acc460921a44c421e57d8e9df140c6dc05a22f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/handsontable/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
```

Comment:
This is missing dependencies; see: https://github.com/handsontable/handsontable/blob/master/package.json#L57

Reply:
For posterity: not actually an issue, see https://github.com/conda-forge/staged-recipes/wiki/Recipes-that-bundle-javascript-libraries.